src/main/webapp/app/exercises/shared/plagiarism/exercise-update-plagiarism/exercise-update-plagiarism.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="form-group col-auto border border-warning p-3">
+<div class="form-group col-auto border border-warning p-3" *ngIf="exercise.plagiarismDetectionConfig">
```

Comment:
I think this introduces false impression that `exercise.plagiarismDetectionConfig` may not be included for some exercises as this level